doc/apple2.sgml

```diff
@@ -334,6 +334,9 @@ usage.
 <item>rebootafterexit
 <item>ser_apple2_slot
 <item>tgi_apple2_mix
+<item>detect_iigs
+<item>get_iigs_speed
+<item>set_iigs_speed
```

Comment:
I'm not sure, if desirable to list those functions here. Doesn't it imply that they become available by including `apple2[enh].h`?

Reply:
Indeed, kind of. And on the other hand, I should have added mktime_dt and gmtime_dt there.

Reply:
What do you think of that change?
I feel like it's good to mention those in the all-things-apple2 page, but it's, indeed, better to split those three from the ones defined in apple2.h.